docs/rules/no_console_log.md

```diff
@@ -0,0 +1,29 @@
+# no_console_log
+
+Disallows the use of `console.log`.
+
+Oftentimes, developers are guilty of committing `console.log` statements accidentally, left in particularly after debugging. Moreover, using `console.log` in code may leak sensitive information to the output or clutter the console with unnecessary information. This rule helps maintain clean and secure code by disallowing the use of `console.log`.
+
+### Invalid
+
+```typescript
+console.log("Debug message");
+
+if debug { console.log("Debugging"); }
+
+function log() { console.log("Log"); }
+```
+
+### Valid
+
+```typescript
+let foo = 0;
+
+const bar = 1;
```

Comment:
not sure what these are doing here

Reply:
Sorry, I can take those out - they were just to show that any other line of code that does not contain `console.log` is valid. Fixed with [this commit](https://github.com/denoland/deno_lint/pull/1189/commits/f21a23064e0fb5195390d33fa4307cf305c6a74f)